### PR TITLE
docs: update `Namespaces` page & fix minor typos

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

This PR updates the Namespaces page to make it more consistent and readable, and updates a few typos in relation to a few missed words in [this PR](https://github.com/gnolang/gno/pull/2514). It renames the Namespaces page to `Package Paths`.

Matching docs PR: https://github.com/gnolang/docs.gno.land/pull/50

<details><summary>Contributors' checklist...</summary>

- [x] Added new tests, or not needed, or not feasible
- [x] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [x] Updated the official documentation or not needed
- [x] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [x] Added references to related issues and PRs
- [x] Provided any useful hints for running manual tests
- [x] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

File: docs/concepts/testnets.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This page documents all gno.land testnets, what their properties are, and how
 they are meant to be used. For testnet configuration, visit the 
 [reference section](../reference/network-config.md).
 
-Gno.land testnets are categorized by 4 main points:
```
Comment (Kouteki) [inline docs/concepts/testnets.md:11]: Isn't the brand guideline that `gno.land` at the start of the sentence is capitalized as `Gno.land`?

Comment (leohhhn) [reply docs/concepts/testnets.md:11]: We discussed internally and the conclusion was that we will write `gno.land` no matter what.

File: docs/concepts/package-paths.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+---
+
+# Package Paths
+
+A package path is a unique identifier for each package/realm. It specifies the 
+location of the package source code which helps differentiate it from others. 
+You can use a package path to:
+
+- Call a specific function from a package/realm. (e.g using `gnokey maketx call`)
+- Import it in other packages/realms.
```
Comment (deelawn) [inline docs/concepts/package-paths.md:12]:
```suggestion
- Import it from other packages/realms.
```

File: docs/concepts/package-paths.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+Here's a breakdown of the structure of a package path:
+
+- Domain: The domain of the blockchain where the package is deployed. 
+Currently, only `gno.land/` is supported.
+- Type: Defines the type of package.
+    - `p/`: [Package](packages.md)
```
Comment (deelawn) [inline docs/concepts/package-paths.md:19]:
```suggestion
    - `p/`: [Pure Package](packages.md)
```

File: docs/concepts/package-paths.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  - Can only contain alphanumeric characters.
+  - No special characters are allowed (except underscore).
+  - Cannot consist solely of underscores. A namespace must have at least one 
+allowed alphanumeric character.
+  - Cannot start with a number. A namespace must begin with a letter.
+  - Cannot end with a trailing slash (`/`).
+
+:::warning Namespaces on gno.land testnets
+
+This feature is currently only enabled on the [Test4 testnet](./testnets.md#test4).
```
Comment (deelawn) [inline docs/concepts/package-paths.md:45]: Maybe reword this to clarify it is enabled on all networks from test4 and beyond, excluding portal loop. This way it won't have to be updated when we launch (test5? 😅 ) mainnet.

File: docs/concepts/package-paths.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+Other networks, such as the Portal Loop, do not have this feature enabled.
+
+:::
+
+## Registering a namespace
+
+To register a namespace, you need to use the `r/demo/users` realm. It allows
+users to register a username for a fee of 200 `GNOTs`, which is in turn used as
```
Comment (deelawn) [inline docs/concepts/package-paths.md:53]:
```suggestion
users to register a username, for a fee of 200 `GNOTs`, which is in turn used as 
```
Makes it clear that the username is the thing being used as a reference for the namespace of the user.

Comment (deelawn) [reply docs/concepts/package-paths.md:53]: Also, I've not typed out a lot of digital currency amounts in markdown, but it seems like `200 GNOT` seems more natural than 200 `GNOTs`. I don't know of any other digital tokens where people use the plural of the token's symbol to reference more than one -- 5 "eth", 10 "sol", 7 "atom", etc.

File: docs/concepts/package-paths.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+For example, the following `gnokey` command will register `patrick` as the username
+for the address `mykey`.
+
+```bash
+$ gnokey maketx call -pkgpath gno.land/r/demo/users \
+    -func Register \
+    -gas-fee 1000000ugnot -gas-wanted 2000000 \
+    -broadcast \
+    -chainid=test4 \
+    -remote https://rpc.test4.gno.land
```
Comment (deelawn) [inline docs/concepts/package-paths.md:73]:
```suggestion
    -remote https://rpc.test4.gno.land \
```

File: docs/concepts/package-paths.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+Make sure to use the proper chain ID and remote when interacting with a 
+gno.land network. Check out the [Network Configurations](../reference/network-config.md)
+page for a list of all available endpoints.
+
+:::
+
+After successful registration, you can add a package under your registered namespace.
+
+## Anonymous Namespace
```
Comment (deelawn) [inline docs/concepts/package-paths.md:90]: I wouldn't describe the namespace as anonymous -- there is an address associated with it. Perhaps "Eponymous Namespace" is more fitting here.

File: docs/concepts/package-paths.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+```bash
+$ gnokey maketx addpkg \
+    --pkgpath "gno.land/r/g1zmgrq5y2vxuqc8shkuc0vr5dj23eklf2xr720x/myrealm" \
+    --pkgdir "." \
+    --deposit 100000000ugnot \
+    --gas-fee 1000000ugnot \
+    --gas-wanted 2000000 \
+    --broadcast \
+    --chainid test4 \
+    -remote https://rpc.test4.gno.land
```
Comment (deelawn) [inline docs/concepts/package-paths.md:107]:
```suggestion
    --remote https://rpc.test4.gno.land \
```
Also when we have blocks of multiline commands in the same doc like this, it would be good to keep the flags consistent -- one dash vs two for example. You and I know there is no difference, but it might be confusing to a newbie just getting started with command line tools.

**Review by deelawn — COMMENTED** [review]
Solid changes. A few things should be fixed and I'd like some input on how we should be referring to packages, pure packages, and realms -- and do so consistently. Of course, that might be out of scope for this PR so we could always punt that particular item to another issue.

**Comment by leohhhn** [comment]
Will be reusing this for the upcoming revamp. Closing here.
